sklearn/calibration.py

```diff
@@ -22,7 +22,7 @@
 from .utils.validation import check_is_fitted
 from .isotonic import IsotonicRegression
 from .svm import LinearSVC
-from .cross_validation import _check_cv
+from .model_selection.partition import _check_cv
```

Comment:
@jnothman Do you think such internal imports should also be routed to the top-level `.model_selection`?